File: lua/test/exists_test.lua
-- HubspotCommunicationPreferences SDK exists test

local sdk = require("hubspot-communication-preferences_sdk")

describe("HubspotCommunicationPreferencesSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// HubspotCommunicationPreferences SDK exists test

require_once __DIR__ . '/../hubspotcommunicationpreferences_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = HubspotCommunicationPreferencesSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}
